test/test.js

```diff
@@ -0,0 +1,81 @@
+var assert      = require('assert');
+var should      = require('should');
```

Comment:
@mptap it looks like line 2 is throwing an error:
```
> mocha
(node:4228) [DEP0006] DeprecationWarning: child_process: options.customFds option is deprecated. Use options.stdio instead.
module.js:538
    throw err;
    ^
Error: Cannot find module 'should'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/test/test.js:2:19)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at /home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:184:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:181:14)
    at Mocha.run (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:393:31)
    at Object.<anonymous> (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/bin/_mocha:380:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
npm ERR! Test failed.  See above for more details.
```

are we using this module in this test? Might be ok to just remove the import?